server/src/test/java/org/elasticsearch/rest/RestCompatibleVersionHelperTests.java

```diff
@@ -24,6 +24,16 @@
     int PREVIOUS_VERSION = RestApiVersion.current().major - 1;
     int OBSOLETE_VERSION = RestApiVersion.current().major - 2;
 
+    // public void testCompatibleOnUnkownMediaType() {
+    // RestRequest.MediaTypeHeaderException e = expectThrows(
+    // RestRequest.MediaTypeHeaderException.class,
+    // () -> requestWith(acceptHeader("wrong/type;compatible-with=7"), contentTypeHeader(null), bodyPresent())
+    // );
+    // assertThat(
+    // e.getMessage(),
+    // equalTo("Unknown media type Accept=wrong/type;compatible-with=")
+    // );
+    // }
```

Comment:
This part right here seems off. Perhaps accidentally committed? WIP that was meant to be removed?

Reply:
ah yes, initially I thought I can include validation for unknown media types before parsing versions, but this is far beyond the scope of this PR.